obkvrpc/rpc_client.go

```diff
@@ -121,18 +129,14 @@ func (c *RpcClient) Execute(ctx context.Context, request protocol.ObPayload, res
 
 	connection, index = c.connectionPool.GetConnection()
 	if connection == nil {
+		// maybe this connection has been disconnected，we need to reconnect it.
```

Comment:
I guess we are using a "user thread"(which will hang the user task) to re-create the connection. Maybe we should do it in the background.

Reply:
maybe we can do it later